app/controllers/api/v1/organizations_controller.rb

```diff
@@ -21,11 +21,7 @@ def check_sync
   end
 
   def update
-    if organization.update_attributes(update_params)
-      render json: { response: 'Updated' }, status: 200
-    else
-      render json: { response: 'Bad request', errors: @user.errors.messages }, status: 400
-    end
+    respond_with organization.update_attributes!  update_params
```

Comment:
⚠️ **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Rails/ActiveRecordAliases: Use update! instead of update_attributes!.

Reply:
buenooo